.gitignore

```diff
@@ -36,3 +36,6 @@ tmp/
 
 /priv/plts/*.plt
 /priv/plts/*.plt.hash
+
+# Vim temporary files
+*.swp
```

Comment:
I don't mind this addition at all, but since you're considering a career in dev, I'll give you this feedback: usually files that need to be ignored because of your specific tooling preferences don't go in the project's `.gitignore`, but your own global `.gitignore`. See: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

My personal `.gitignore` looks like this (macOS and RubyMine stuff)
```
$ cat ~/.gitignore
.DS_Store
.AppleDouble
.LSOverride

# Icon must end with two \r
Icon

# Thumbnails
._*

# Files that might appear in the root of a volume
.DocumentRevisions-V100
.fseventsd
.Spotlight-V100
.TemporaryItems
.Trashes
.VolumeIcon.icns

# Directories potentially created on remote AFP share
.AppleDB
.AppleDesktop
Network Trash Folder
Temporary Items
.apdisk

.idea
*.iml
```

Reply:
OMG is so obvious. Of course I should have been doing this for years!  :hurtrealbad: :hurtrealbad: :hurtrealbad: